File: Java/day26.java
// Enter your code here. Read input from STDIN. Print output to STDOUT
import java.util.*;

public class day26
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] actual = new int[3];
        int[] expected = new int[3];
        int fine = 0;
        int late_days = 0;

        for (int i = 0; i < 3; i++)
            actual[i] = sc.nextInt();
        for (int i = 0; i < 3; i++)
            expected[i] = sc.nextInt();

        if (expected[2] >= actual[2]) {
            if (expected[1] >= actual[1]) {
                if (expected[0] >= actual[0])
                    fine += 0;
                else {
                    late_days = actual[0] - expected[0];
                    fine += 15 * late_days;
                }
            } else {
                fine += 500 * (actual[1] - expected[1]);
            }
        } else {
            fine += 10000;
        }

        System.out.println(fine);
    }
}

File: Scala/day26.scala
object Solution {
  def main(args: Array[String]) {
    // Write your code here
    val sc = new java.util.Scanner(System.in)
    var actual = new Array[Int](3)
    var expected = new Array[Int](3)
    var fine = 0

    for (i <- 0 to 2) actual(i) = sc.nextInt()
    for (i <- 0 to 2) expected(i) = sc.nextInt()

    if (expected(2) >= actual(2)) {
      if (expected(1) >= actual(1)) {
        if (expected(0) >= actual(0)) fine += 0 else fine += 15 * (actual(0) - expected(0))
      } else fine += 500 * (actual(1) - expected(1))
    } else fine += 10000

    println(fine)
  }
}
